modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+The Container Storage Interface (CSI) allows {product-title} to expand storage volumes after they have already been created.
+
+Support for expanding CSI volumes is enabled by default. However, to use CSI volume expansion, you must have a specific CSI driver that supports volume expansion.
```

Comment:
Could say "{product-title} supports CSI volume expansion by default," but that's just nitpicking.